includes/bin/runghostinspectorsuite

```diff
@@ -96,8 +99,8 @@ echo "Using JSON params: $GI_PARAMS_JSON"
 # Execute the suite
 set +e
 OUTPUT=$(GI_API_KEY=$GI_API_KEY & /bin/run-suite.js $GI_SUITE "$GI_PARAMS_JSON")
-
 PASSING=$?
+# echo $OUTPUT
```

Comment:
Is this commented out because it's for debugging?

Reply:
Yeah, I can take it out if you want

Reply:
Nah, it's fine. Just wasn't sure if you meant to have it there.